transformations/GermanGenderSwap/transformation.py

```diff
@@ -0,0 +1,2433 @@
+# -*- coding: utf-8 -*-
+
+import random
+
+from typing import List
+from checklist.editor import Editor
+
+from interfaces.SentenceOperation import SentenceOperation
+from tasks.TaskTypes import TaskType
+#TOP 1000 MOST COMMON GERMAN MALE AND FEMALE NAMES,
+names={1: {0: 'Mary',
```

Comment:
Can you move this to a separate json file?

Reply:
Moved it!